syntax = "proto3";
import "SerializableOperator.proto";
package NES;

/*
The serializable wrapper definition for query plan
 */
message SerializableQueryPlan{
  repeated SerializableOperator rootOperators = 1;
  uint64 queryId = 2;
  uint64 querySubPlanId = 3;
}